src/xgov-dapp/src/features/vote-creation/RoundInfo.tsx

```diff
@@ -67,6 +69,8 @@ function validateProposalCsv(value: string, ctx: z.RefinementCtx) {
       }
     })
   })
+
+  return Papa.unparse(parsed as unknown as UnparseObject<Proposal>)
```

Comment:
```suggestion
return Papa.unparse({ data: parsed.data, fields: parsed.meta.fields || [] })
```
According to https://www.papaparse.com/docs#unparse, it expects an object with `data` and `fields` properties.
@PhearZero what do you think about the above suggestion?

Reply:
The main issue I have with changing the pattern is the additional object created causing more work for the GC. The documentation shows passing the parsed object directly to the unparse method.